### PR TITLE
chore: update to 4.29

### PR DESCRIPTION
**Pull request by andygup**
@annelfitz can you take a look? 

/cc @bsvensson

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,289 +1,221 @@
...
-- Enhanced the styling of empty [GroupLayers](https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-GroupLayer.html) in the [LayerList](https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-LayerList.html) by adding a dashed outline and the text "There are currently no items to display."
+- TBD
 
 ## Deprecations
 
-The following are deprecated and will be removed in a future release. For anything deprecated in 4.26 and earlier, additional information and links are in the [release notes](https://developers.arcgis.com/javascript/latest/release-notes/#deprecated-classes-properties-methods-events).
+The following are deprecated and will be removed in a future release. For anything deprecated in 4.27 and earlier, additional information and links are in the [release notes](https://developers.arcgis.com/javascript/latest/release-notes/#deprecated-classes-properties-methods-events).
```
Comment (annelfitz) [inline CHANGELOG.md:29]: should this be 4.28?
```suggestion
The following are deprecated and will be removed in a future release. For anything deprecated in 4.28 and earlier, additional information and links are in the [release notes](https://developers.arcgis.com/javascript/latest/release-notes/#deprecated-classes-properties-methods-events).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,289 +1,221 @@
...
 
-- For local builds, [Webpack](https://webpack.js.org/) versions prior to `5.84.0` are deprecated at 4.27. This is related to bug fixes in Webpack.
+- The allowAttachments property within Editor.layerInfos is deprecated at 4.25. Use either attachmentsOnCreateEnabled or attachmentsOnUpdateEnabled instead.
+- The "non-metric" possible value for ScaleBar.unit is deprecated at 4.27. Please use "imperial" instead.
+- As of 4.27, when building new custom widgets it is no longer recommended to subclass or extend esri/widgets/Widget. Instead, use DOM elements such as DIVs, web components or JavaScript framework components in combination with the Calcite Design System. More information about creating a custom UI is available in the [Custom UI basics](https://developers.arcgis.com/javascript/latest/custom-ui/) Guide topic.
```
Comment (annelfitz) [inline CHANGELOG.md:33]: why are these outside of the "complete list" below??

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,289 +1,221 @@
...
+-	FeatureTableViewModel.clearSelection deprecated since version 4.25. Use highlightIds.removeAll() instead.
+-	FeatureTableViewModel.highlightOnRowSelectEnabled deprecated since version 4.25. Use highlightEnabled instead.
+-	FeatureTableViewModel.selectRows deprecated since 4.25. Use highlightIds.add() instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
```
Comment (annelfitz) [inline CHANGELOG.md:156]: Can you copy the list of deprecations from the production release notes? Seeing a lot of duplicates here that are not in the production release notes list

**Review by annelfitz — CHANGES_REQUESTED** [review]
Looks pretty good - but can you clean up the deprecations list by replacing it with the list from the production release notes?

File: CHANGELOG.md (hunk 3 above)
Comment (andygup) [reply CHANGELOG.md:156]: Wow, I see that now. I'll fix it. I created the PR several weeks ago and clearly didn't look close enough. I also see comments in there like this:

-	Popup.maxInlineActions deprecated TODO: DOCUMENT

Comment (andygup) [reply CHANGELOG.md]: Production deprecations have been re-added. Sorry about that.

**Pull request merged**